### PR TITLE
EOS-26859 - Include optional parameters in support bundle script

### PR DESCRIPTION
**Pull request by shahidSTX**
Signed-off-by: Mohammed Shahid <mohammed.shahid@seagate.com>

# Problem Statement
- As per new design include optional parameters in support bundle script.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/s3-support-bundles/s3_bundle_generate.sh [hunk 1]
```diff
@@ -31,18 +31,32 @@ where:
...
+    -t) bundle_path="$2";shift 2;;
+    -c) confstore_url="$2";shift 2;;
+    -s) services="$2";shift 2;;
+    --duration) duration="$2";shift 2;;
+    --size_limit) size_limit="$2";shift 2;;
+    --binlogs) binlogs="$2";shift 2;;
+    --coredumps) coredumps="$2";shift 2;;
+    --stacktrace) stacktrace="$2";shift 2;;
+    --) shift;break;;
+    *) usage;exit 1;;
```
Comment (nileshgovande) [inline scripts/s3-support-bundles/s3_bundle_generate.sh:59]: Update usage @shahidSTX

Comment (nileshgovande) [reply scripts/s3-support-bundles/s3_bundle_generate.sh:59]: Also - 'where are we differentiating 'mandatory' and 'optional' parameters

Comment (shahidSTX) [reply scripts/s3-support-bundles/s3_bundle_generate.sh:59]: The check for mandatory parameters remains the same, using -z flag.

Comment (shahidSTX) [reply scripts/s3-support-bundles/s3_bundle_generate.sh:69]: Usage updated.

**Review by vimalraghwani20 — APPROVED** [review]

**Pull request merged**